Add ability to queue weapon changes mid-attack

exscape opened the pull request:
When attempting to switch weapons in the middle of a fight, the game will by default
simply ignore your request if you're currently attacking.
This patch allows to queue them and switch ASAP when the attack is finished.
Supports swiching to multiple items, e.g. sword and shield.

Also queues hiding weapons (default R) if pressed mid-attack.

nxPublic commented:
Seems like your local repository was a little out of date, sorry that I directly push to the main repo half a million times a day sometimes. Feel free to contact me on Discord so I can assign you the GitHub contributor role 👍

exscape commented:
There! I think I fixed it.

However, I did spot an error in vplusconfig.json from one of the recent commits that is also fixed here. Not sure if I should move that out or something as it doesn't really have anything to do with my edits.

healiha commented:
> However, I did spot an error in vplusconfig.json from one of the recent commits that is also fixed here. Not sure if I should move that out or something as it doesn't really have anything to do with my edits.

Don't worry about making another PR, we always welcome those kind of fixes even if they're not directly part of the initial PR :)
